src/core/eme/utils/persistent_sessions_store.ts

```diff
@@ -137,6 +178,21 @@ export default class PersistentSessionsStore {
     this._save();
   }
 
+  deleteLast(sessionsToDelete : number) : void {
+    log.debug(`EME-PSS: Deleting last ${sessionsToDelete} sessions.`);
+    if (sessionsToDelete <= 0) {
+      return;
+    }
+    if (sessionsToDelete <= this._entries.length) {
+      this._entries.splice(0, sessionsToDelete);
+    } else {
+      log.warn("EME-PSS: Asked to remove more information that it contains",
+               sessionsToDelete,
+               this._entries.length);
+      this._entries = [];
+    }
+  }
```

Comment:
I think we're missing a call to `_save` here. And also, should this function be called deleteLastSessions or deleteLastOnes to clarify the fact that there may be several sessions to delete but not only the last one ?

Reply:
You're right on both counts. I'll do it.

Reply:
Done